kord-extensions/src/main/kotlin/com/kotlindiscord/kord/extensions/commands/Command.kt

```diff
@@ -99,6 +144,15 @@ public abstract class Command(public val extension: Extension) : Lockable, KordE
             extension.bot.send(event)
         }
 
+    /** Returns the full hierarchy of names until this command, locale can be used for translation. **/
+    public open fun getFullName(locale: Locale? = null): String = name
+
+    internal open suspend fun onSuccessUseLimitUpdate(
```

Comment:
Maybe this method should be replaced with an event listener on kordex command execution events like `CommandFailedWithExceptionEvent` and `CommandSucceededEvent`, but these events are too generic at the moment, they don't provide what user, channel, guild it ran inside.
+ So currently the point of this method is to serve as an eventlistener with plenty context information to give to the useLimiters. 

I could change this but it would probably be breaking changes to many of CommandInvocationEvent types.